conda_build/conda_interface.py

```diff
@@ -4,123 +4,93 @@
 
 from functools import partial
 import os
-from importlib import import_module
+from importlib import import_module  # noqa: F401
 import warnings
 
-from conda import __version__ as CONDA_VERSION
+from conda import __version__ as CONDA_VERSION  # noqa: F401
 
-CONDA_VERSION = CONDA_VERSION
-
-
-def try_exports(module, attr):
-    # this assumes conda.exports exists, so only use for conda 4.3 onward
-    try:
-        return getattr(import_module('conda.exports'), attr)
-    except AttributeError:
-        return getattr(import_module(module), attr)
-
-
-try:
-    # This monkey patch is addressed at #1825. The ensure_use_local is an outdated vestige
-    #   and no longer has any relevant effect.
-    import conda.cli.common
-    conda.cli.common.ensure_use_local = lambda x: None
-except ImportError:
-    # no need to patch if it doesn't exist
-    pass
-
-# All of these conda's are older than our minimum dependency
-conda_43 = True
-conda_44 = True
-conda_45 = True
-conda_46 = True
-conda_47 = True
-conda_48 = True
-conda_411 = True
```

Comment:
These are public attributes and were removed if I am not mistaken?

@kenodegard

Reply:
@jezdez [responded on Matrix](https://matrix.to/#/!TfaKrPTflzGMvGhxtd:gitter.im/$uPoHHpqtRvcoPFiAT3GbDgkAP-Qtte7jvkUsXjc3ecE?via=gitter.im&via=matrix.org&via=salt-rock-lamp.ems.host), reposting here for visibility:

> Sorry for the mess @beckermr, seems like it wasn't clear that this was used elsewhere, for better or worse. I'll kick off the discussion to apply [CEP 8](https://github.com/conda-incubator/ceps/blob/main/cep-8.md) and [9](https://github.com/conda-incubator/ceps/blob/main/cep-9.md) on conda-build next week to make sure we have some lead-time when removing things like this.

Reply:
yeah, this broke the concourse (anaconda CI) command-line tool